infiniband-exporter.py

```diff
@@ -61,6 +61,14 @@ def __init__(self, can_reset_counter, input_file, node_name_map):
                 'severity': 'Error',
                 'bits': 16,
             },
+            # detailed description of xmitDiscards: (Head of Queue) timeout https://community.mellanox.com/s/article/howto-prevent-infiniband-credit-loops
+            # possibly interesting to also monitor a lot of similar performance metrics that have been introduced in the kernel at the same time https://patchwork.kernel.org/project/linux-rdma/patch/0F3DDD57-C1A9-4304-A20F-7027BF8F590B@llnl.gov/
```

Comment:
please drop comment in code and create an issue on the project page of the maintainer